FluentTerminal.App.ViewModels/TerminalViewModel.cs

```diff
@@ -394,6 +400,21 @@ private void CloseSearchPanel()
             FocusTerminal();
         }
 
+        private bool CanExecuteCommand<T>(T a)
+        {
+            return CanExecuteCommand();
+        }
+
+        private bool CanExecuteCommand()
+        {
+            return (Initialized == true && _disposalRequested == false);
+        }
+
+        private async void CloseTab()
```

Comment:
this looks dangerous, I think we should use `AsyncCommand` like for `EditTitleCommand`

Reply:
@felixse will it be fine just to revert back to anonymous definition on CloseCommand `async () => await TryClose().ConfigureAwait(false)` ? P.S. Could you please give more general details why it might be dangerous, thanks!

Reply:
@felixse hmmm ... if I revert back to `CloseCommand = new RelayCommand(async () => await TryClose().ConfigureAwait(false), CanExecuteCommand);` I'm starting to receive again crashes from random places (if to close fast just opened tabs).

Reply:
the anonymous action he had before was bad as well. Can you try with `AsyncCommand` and `async Task CloseTab()` signature?

Reply:
Sure, going to give it a try

Reply:
@felixse seems AsyncCommand works good, have pushed new commit with it, thanks